tests/tests_unit/test_base.py

```diff
@@ -370,6 +370,25 @@ def test_use_method_which_requires_cognite_client__client_not_set(self):
         with pytest.raises(CogniteMissingClientError):
             mr.use()
 
+    @pytest.mark.dsl
+    def test_to_pandas_method(self):
+        import pandas as pd
+
+        from cognite.client.data_classes import Asset, AssetList
+
+        obj = AssetList([Asset(external_id=f"ext-{i}", name=f"name-{i}") for i in range(5)])
+
+        result_df = obj.to_pandas()
```

Comment:
This does *not actually call your code*, it will just call dump on each asset. I suggest you just create one test asset with quite a few of the accepted parameter:
```
1. external_id
2. name
3. parent_id
4. parent_external_id
5. description
6. data_set_id
7. metadata
8. source
9. labels
10. geo_location
11. id
12. created_time
13. last_updated_time
14. root_id
15. aggregates
```
Apart from that, the test looks great!